packages/waku/src/router/create-pages.ts

```diff
@@ -350,6 +390,10 @@ export function createPages<
     },
     async (id, { unstable_setShouldSkip }) => {
       await configure();
+      if (id === 'root') {
+        unstable_setShouldSkip();
```

Comment:
Shouldn't it be `unstable_setShouldSkip([])` if it's static?

Reply:
like this?

```ts
        if (rootItem?.render === 'dynamic') {
          unstable_setShouldSkip();
        } else unstable_setShouldSkip([]);
```

Reply:
Yeah.